src/cls/IPM/ResourceProcessor/Default/Document.cls

```diff
@@ -172,6 +172,26 @@ Method OnPhase(pPhase As %String, ByRef pParams, Output pResourceHandled As %Boo
                 Set tSC = $$$OK
                 Quit
               }
+              If ..ResourceReference.Deploy {
+                If tVerbose {
+                  Write !, "Skipping deployed resource: " _ ..ResourceReference.Name
```

Comment:
I'm a bit confused by this. Is the resource getting skipped or is it just getting loaded from a studio project so regular handling is skipped?